cmd/ebrelayer/txs/parser.go

```diff
@@ -42,8 +39,14 @@ func ParsePayload(valAddr sdk.ValAddress, event *events.LockEvent) (ethbridgeTyp
 		return witnessClaim, errors.New("empty recipient address")
 	}
 
+	// Symbol formatted to lowercase
+	symbol := strings.ToLower(event.Symbol)
+	if symbol == "eth" && event.Token != common.HexToAddress("0x0000000000000000000000000000000000000000") {
+		return witnessClaim, errors.New("symbol \"eth\" must have null address set as token address")
```

Comment:
Can we add an additional check of this that runs under consensus within the module, ie, during ValidateBasic of peggy\x\ethbridge\types\msgs.go

Reply:
https://github.com/swishlabsco/peggy/pull/4/commits/be6ce9d49dac7195bbcf2bf3b5260ebd2afc5715